package.json

```diff
@@ -32,7 +32,8 @@
   "dependencies": {
     "canvas": "^2.6.0",
     "chalk": "^2.4.2",
-    "fs-extra": "^7.0.1"
+    "fs-extra": "^7.0.1",
+    "gm": "^1.23.1"
```

Comment:
I try to prevent using _non standard_ environment modules like `imagemagick` and `graphicsmagick`. Not every user/customer has this standard installed in their CI pipeline which makes this break in pipelines.

ResembleJS also provides a `ignoreColors` option, doesn't that work?

Reply:
I will have to take a look and I will look at commments soon!

Reply:
OK so `ignoreColors` does not work for me, that basically will compare at grayscale.

The usage of this solves is reading "painted on" captions from a video frame where the video frame may have slight differences.

Our app is a testbed for our video player, in that player, there are 608 captions painted on, over video. In order to validate them, we need to extract captions. However, since there are small changes to the background image even ignoring colors doesn't help. What this does is take any color and floor it to black (not grays), leaving a pure black image with white text to compare.

Reply:
Also, these libraries are only looked for and used if this option is enabled... I understand non-standard libraries but if you use this option you will need to install the required components.